dissect/target/plugins/os/windows/sru.py

```diff
@@ -322,7 +340,7 @@
 }
 
 
-def transform_app_id(value):
+def transform_app_id(value: Union[bytes, str, None]) -> Union[str, None]:
```

Comment:
```suggestion
def transform_app_id(value: Optional[Union[bytes, str]]) -> Optional[str]:
```

Reply:
Fixed in [71cb7ce](https://github.com/fox-it/dissect.target/pull/472/commits/71cb7ce665adfff82ebdcd0bf606325fbb73b718)